packages/lib/src/prod/shared-production.ts

```diff
@@ -289,16 +289,16 @@ export function prodSharedPlugin(
     },
     renderChunk: function (code, chunk, options) {
       //   process shared chunk
-      const sharedFlag = sharedFileReg.test(basename(chunk.fileName))
+      // const sharedFlag = sharedFileReg.test(basename(chunk.fileName))
       const facadeModuleId = chunk.facadeModuleId
-      const exposesFlag = parsedOptions.prodExpose.some((expose) =>
-        isSameFilepath(expose[1].id, facadeModuleId as string)
-      )
+      // const exposesFlag = parsedOptions.prodExpose.some((expose) =>
+      //   isSameFilepath(expose[1].id, facadeModuleId as string)
+      // )
       const needSharedImport =
         isRemote &&
         parsedOptions.prodShared.length > 0 &&
         chunk.type === 'chunk' &&
-        (sharedFlag || exposesFlag) &&
+        // (sharedFlag || exposesFlag) &&
```

Comment:
It is not a good practice to skip the check altogether ,because only the components of exposes need the shard function, other components that are not related to expose should not use shared

Reply:
yes, it should be handled with more logically here. the related codes were reverted